src/components/Team/proPlayerImages.js

```diff
@@ -0,0 +1,189 @@
+export default ([
```

Comment:
suggest moving this to a manifest.json file in the images directory so that the manifest and images are in one place,

You can then require() the file from there.